scene/2d/particles_2d.cpp

```diff
@@ -235,7 +235,7 @@ _FORCE_INLINE_ static float _rand_from_seed(uint32_t *seed) {
 		s = 0x12345987;
 	k = s / 127773;
 	s = 16807 * (s - k * 127773) - 2836 * k;
-	if (s < 0)
+	if (s < 0)	// but "s" is unsigned ?
```

Comment:
No, it's because it was replaced a first time in a747c8c, and then a second time in 4c90046. I guess both contributors (cc @tagcup) missed that there was code duplication in this class.